tests/Drupal/Commenting/DocCommentLongArraySyntaxUnitTest.inc.fixed

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @file
+ * Test long array syntax in doc comments.
+ */
+
+/**
+ * Long array syntax used.
+ * @code
+ * $table = [
```

Comment:
you did not implement a fixer, so the tests should actually fail because this can never correct itself?

I think we can't implement a fixer for it right now, so this file should be removed.

Reply:
Removed.